### DISCUSSION TITLE
Continuous max CPU usage when NextCloud clients sync data

### DISCUSSION THREAD

**Discussion by DrPleb** [nextcloud/all-in-one#3226] [Questions] [answered]
Hi, I've the latest NextCloud AiO version with HAproxy, the system runs smooth when I used via web, but when i start install 2/3 or more clients, it starts get max CPU usage costantly, getting it unstable and not responding. The server is 8cores, 12GB RAM.

I left 3 clients sync during night and at the morning the server was still on max cpu usage and I have to force it to reboot.

Any ideas? I think that nextcloud clients last version are bugged or the problem is in nextcloud 27...

I have also a Nextcloud 25 (installation without aio, the manual standard one) with 30 clients installed and syncing without problems

![image](https://github.com/nextcloud/all-in-one/assets/132892195/b006e910-1c47-4a3d-9a80-643a811a230b)

**Comment by szaimen** [comment, marked as answer]
Hi, can you disable collabora for a test and check if that helps?

**Reply by DrPleb** [reply]
Hi, I tried but nothing changes. I tried to upload files to NextCloud by Rclone with webdav. It works better and the CPU is fine if the number of multiple upload streams is 4, if I try with 8 or more the CPU suffers a lot. With 6 vCPU and 8 upload streams it was unstable for 5/10 minutes, and then it stabilized. I think that the nextcloud client uses too much streams and crashes the CPU of the server, and if I do this from multiple clients, the server goes unreachable.

Now I'm uploading about 120GB with 8 streams and the CPU used is about 9/10 points on Nextcloud system interface.

I don't know if this is a problem only of the Nextcloud AiO or is normal...

Any suggestion?

**Reply by szaimen** [reply]
> I don't know if this is a problem only of the Nextcloud AiO or is normal...

Not sure either. Probably good to limit the amount of streams anyway.
